src/comp_loinc/analysis/snomed.py

```diff
@@ -0,0 +1,80 @@
+"""Builds a representation of SNOMED in OWL functional syntax"""
```

Comment:
Creates `snomed-unreasoned.ofn`. Then, the makefile creates `snomed-reasoned.ofn`.

Used as a prerequisite for LOINC-SNOMED representation (WIP).